accessibility/common-remediations.md

```diff
@@ -690,3 +692,60 @@ We _don’t_ do this:
 <span>Visit this location</span>
 <span>This is a level 2 heading</span>
 ```
+
+## Focus management
+
+### No focus management
+
+#### What's the problem?
+
+Programmatic focus is absent. You may have written an SPA that fails to handle focus management automatically, or you may have neglected to implement programmatic focus on a component that changes the user's context (e.g. a modal dialog).
+
+#### Who's affected by the problem?
+
+* Screen readers
+* Keyboard users
+
+#### Why's it a problem?
+
+"Focus management" is the practice of setting the user's current focus programmatically. 
+
+Most SPA frameworks fail to handle this automatically, and you need to implement it yourself. When a user activates a control to trigger a route change, the view visually updates. When you don't implement focus management, screen reader users are not made aware that the context has changed, and are left guessing whether or not activating the control did anything. Keyboard users may be able to see the visual changes, but they're left with their keyboard focus still on the trigger control. The control is no longer visible on the screen, and may require additional, unnecessary work from the keyboard user to get reoriented. 
```

Comment:
Should this comma be removed? as it is followed by 'and'.

Reply:
```suggestion
Most SPA frameworks fail to handle this automatically, and you need to implement it yourself. When a user activates a control to trigger a route change, the view visually updates. When you don't implement focus management, screen reader users are not made aware that the context has changed, and are left guessing whether or not activating the control did anything. Keyboard users may be able to see the visual changes, but they're left with their keyboard focus still on the trigger control. The control is no longer visible on the screen and may require additional, unnecessary work from the keyboard user to get reoriented. 
```